data/tree/iterator/internal/dfb2fd20-66a1-4163-a344-cc89d1d0fd4b.json

```diff
@@ -9,26 +9,30 @@
   "interface": {
     "inputs": {
       "f9494c81-42f8-41eb-8ca3-549f2c2f5a99": {
-        "type": "any",
+        "type": "{\"tree\": {string: any}, \"query\": (string or number or {\"type\": \"wildcard\"} or {\"type:\"options\", \"options\": (string or number)[]})[]}",
```

Comment:
or maybe refer on the parent port to the type of the child's port with the new `of` keyword to avoid double documentation? (not sure if possible and also not sure whether it is a good for the user at the moment...)

Reply:
For the time being I think it's fine if these types are duplicated, as we'll use named types for them in the future: `data/tree/Tree`, `data/tree/Query`. The tree type does need to be updated though.